user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementDetailsComponent.java

```diff
@@ -144,12 +156,14 @@ private void resetTabsInTabsheet() {
 
   private void addMeasurementTab(GridLazyDataView<?> gridLazyDataView) {
     if (gridLazyDataView.getItem(0) instanceof ProteomicsMeasurement) {
```

Comment:
What happens when the data view does not contain items? IndexOutOfBoundException?

Reply:
it's only called once the views are initialized, but I added a check anyway